Source/Fuse.Common/Internal/iOSSystemFont.uno

```diff
@@ -329,6 +425,110 @@ namespace Fuse.Internal
 			throw new Exception("iOSSystemFont: No matching style in " + fileName);
 		}
 
+
+
+		internal static ObjC.Object GetMatchingSystemFontDescriptor(Fuse.SystemFont.Style fontStyle, Fuse.SystemFont.Weight fontWeight, Fuse.SystemFont.Design fontDesign, int fontSize)
+		{
+			string selectedFontStyle = "normal";
+			string selectedFontWeight = "normal";
+			string selectedFontDesign = "default";
+
+			switch(fontStyle) {
```

Comment:
Code style nits (three occurrences in this method):
- Add space between `switch` and `(`
- Move curly braces to their own lines